### PR TITLE
Comments audio example

### PR DESCRIPTION
**Pull request by CTNicholas**
**TODO**
- [ ] Make pretty @pierrelevaillant 
- [ ] Thumbnails for github @pierrelevaillant 
- [ ] Thumbnails for liveblocks.io @pierrelevaillant 
  - https://github.com/liveblocks/liveblocks.io/pull/1884

### CODE REVIEW

File: examples/nextjs-comments-audio/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  }, []);
+}
+
+export function useSkipToListener(callback: (timePercentage: number) => void) {
+  useEffect(() => {
+    function handler(event: SkipToEvent) {
+      callback(event.detail.timePercentage);
+    }
+
+    window.addEventListener(SKIP_TO_EVENT_NAME, handler as any);
```
Comment (marcbouchenoire) [inline examples/nextjs-comments-audio/src/utils.ts:22]: Is there no way to not `as any`?

File: examples/nextjs-comments-audio/src/components/ThreadsTimeline.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import { useSkipTo } from "@/utils";
+import { ThreadData } from "@liveblocks/core";
+import { ClientSideSuspense } from "@liveblocks/react";
+import { Comment } from "@liveblocks/react-comments/primitives";
+import { CSSProperties, useCallback, useState } from "react";
+import { ErrorBoundary } from "react-error-boundary";
+
+export function ThreadsTimeline() {
+  return (
+    // @ts-ignore
```
Comment (marcbouchenoire) [inline examples/nextjs-comments-audio/src/components/ThreadsTimeline.tsx:15]: What's the TS error here?

**Review by marcbouchenoire — CHANGES_REQUESTED** [review]
Spotted a few typos and left a couple of questions, but otherwise it's looking good!

File: examples/nextjs-comments-audio/src/components/ThreadsTimeline.tsx (hunk 2 above)
Comment (CTNicholas) [reply examples/nextjs-comments-audio/src/components/ThreadsTimeline.tsx:15]: This is a bit of a rabbit hole! If I recall correctly, the only way to fix this is to downgrade `@types/react`, so I'd prefer leave leave in the `ts-ignore`

**Review by marcbouchenoire — APPROVED** [review]

**Pull request merged**